### PR TITLE
Added admin sign in page

### LINKED ISSUE #41: Create 'login' page
fields: username, (required) password (required)
-login button

Use WET for form accessibility

### PR DESCRIPTION
**Pull request by khalidkanaan**

### CODE REVIEW

**Review by mputnik — APPROVED** [review]

File: react-website/src/App.js [hunk 1]
```diff
@@ -15,6 +16,7 @@ function App() {
           <Route index element={<Homepage />}/>
           <Route path="confirmation" element={<Confirmation />} />
           <Route path="form" element={<Form />} />
+          <Route path="admin-sign-in" element={<AdminSignIn />} />
```
Comment (khalidkanaan) [inline react-website/src/App.js]: Can you change this route to something like "admin/sign-in"

**Review by Dweevil — APPROVED** [review]
fantastic work

**Review by jjsoong — APPROVED** [review]

File: react-website/src/App.js (hunk 1 above)
Comment (Justinzoorob) [reply react-website/src/App.js]: if you change this make sure to change the path in the header.js file for the button and the drop down menu.

**Review by Justinzoorob — APPROVED** [review]

**Comment by jjsoong** [comment]
Changed to branch-to-branch merge.
Replaced by #54.

**Pull request closed without merging**